src/foremast/awslambda/awslambda.py

```diff
@@ -161,12 +214,18 @@ def create_function(self, vpc_config):
 
             raise
 
-        LOG.info("Successfully created Lambda function")
+        LOG.info("Successfully created Lambda function and alias")
 
     def create_lambda_function(self):
         """Create or update Lambda function."""
         vpc_config = self._vpc_config()
         if self._check_lambda():
             self.update_function_configuration(vpc_config)
+
+            if self._check_lambda_alias():
```

Comment:
Can we move this `if` lower to avoid nesting?
```python
if _check_lambda():
    update_function()
else:
    create_function()

if _check_lambda_alias():
    update_alias()
else:
    create_alias()
```
It might even make more sense to move the if statements into the `checks()`.
```python
def _check_lambda(self):
    self.vpc_config = self._vpc_config()

    try:
        self.lambda_client.get_function(FunctionName=self.app_name)
        self.update_function_configuration()
    except boto3.exceptions.botocore.exceptions.ClientError:
        self.create_function()

def _check_lambda_alias(self):
    if found:
        self.update_alias()
    else:
        self.create_alias()
```